supertokens_python/recipe/session/interfaces.py

```diff
@@ -602,7 +602,9 @@ class SessionClaimValidator(ABC):
     def __init__(
         self,
         id_: str,
-        claim: SessionClaim[_T],  # pyright: ignore[reportInvalidTypeVarUse]
+        claim: Optional[
+            SessionClaim[_T]  # pyright: ignore[reportInvalidTypeVarUse]
+        ] = None,
     ) -> None:
         self.id = id_
         self.claim = claim
```

Comment:
`self.claim: Option[SessionClaim[Any]] = None`

Reply:
Then you don't need to take claim as an optional input.

Reply:
Done.